search/indexing_api.py

```diff
@@ -669,21 +669,30 @@ def index_run_content_files(run_id, upate_only=False):
         update_only (bool): Update existing index only
 
     """
-    run = LearningResourceRun.objects.get(id=run_id)
-    documents = (
-        serialize_content_file_for_bulk(content_file)
-        for content_file in run.content_files.select_related("run")
-        .prefetch_related("run__content_object")
-        .defer("run__raw_json")
-    )
-    index_items(
-        documents,
-        COURSE_TYPE,
-        upate_only,
-        routing=gen_course_id(
-            run.content_object.platform, run.content_object.course_id
-        ),
-    )
+    run = LearningResourceRun.objects.get(pk=run_id)
+    content_file_ids = run.content_files.values_list("id", flat=True)
+
+    chunk_size = settings.ELASTICSEARCH_DOCUMENT_INDEXING_CHUNK_SIZE
+
+    for index in range(0, len(content_file_ids), chunk_size):
+        ids_chunk = content_file_ids[index : index + chunk_size]
+
+        documents = (
+            serialize_content_file_for_bulk(content_file)
+            for content_file in ContentFile.objects.filter(pk__in=ids_chunk)
+            .select_related("run")
+            .prefetch_related("run__content_object")
+            .defer("run__raw_json")
+        )
+
+        index_items(
+            documents,
+            COURSE_TYPE,
+            upate_only,
```

Comment:
While we're here can we rename this to `update_only`? It doesn't look like there are any calling functions which use this argument as a keyword argument so it should be a simple change hopefully